autoload/neomake.vim

```diff
@@ -476,6 +485,14 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
             endif
         endif
         let status = a:data
+        if has_key(jobinfo, 'exit_callback')
+            try
+                let callback_dict = { 'status': status,
+                                    \ 'name': maker.name,
+                                    \ 'has_next': has_key(maker, 'next') }
```

Comment:
`has_next` is an internal property. I don't think it should be exposed. It's only used in certain cases (mainly when serializing multiple jobs, which isn't done by default).